Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.4
```

Comment:
isn't more configuration needed for this?
https://github.com/hmcts/cmc-claim-store/pull/449/files#diff-3254677a7917c6c01f55212f86c57fbfR3

Reply:
Fixed 🗡